### DISCUSSION TITLE
background image for all pages

### DISCUSSION THREAD

**Discussion by IngoManthey** [QuestPDF/QuestPDF#1059] [Q&A] [answered]
Is it possible, for example, to display a background image on each invoice that has 4 pages?

**Comment by MarcinZiabek** [comment, marked as answer]
Yes :) You can use the watermark slots of the Page component. It is possible to put any content, including images: https://www.questpdf.com/api-reference/page.html#watermark-slots

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
